lib/progress.coffee

```diff
@@ -15,12 +15,17 @@ limitations under the License.
 ###
 
 noop = require('lodash/noop')
-zlib = require('zlib')
-stream = require('stream')
-progress = require('progress-stream')
-rindle = require('rindle')
+once = require('lodash/once')
 utils = require('./utils')
 
+# Lazy-load dependencies to prevent this happening in the browser
+# where this module is not applicable (but still included into the bundle)
+loadDeps = once -> {
```

Comment:
The gain isn't quite the same, because for instance if something only calls `estimate` it would still have to pay the cost to load `progress-stream` (even if it never calls `getProgressStream`)